apps/cowswap-frontend/src/modules/swap/containers/SwapModals/index.tsx

```diff
@@ -2,21 +2,18 @@ import React from 'react'
 
 import { genericPropsChecker } from '@cowprotocol/common-utils'
 
-import { SurplusModalSetup } from '../SurplusModalSetup'
-
 export interface SwapModalsProps {
   showNativeWrapModal: boolean
   showCowSubsidyModal: boolean
 }
 
 export const SwapModals = React.memo(function (props: SwapModalsProps) {
-  const { showNativeWrapModal } = props
-
   return (
     <>
       {/* TODO: Re-enable modal once subsidy is back  */}
       {/*<CowSubsidyModal isOpen={showCowSubsidyModal} onDismiss={closeModals} /> */}
-      {!showNativeWrapModal && <SurplusModalSetup />}
+      {/* TODO: Re-enable modal for displaying when progress bar is closed */}
+      {/*{!showNativeWrapModal && <SurplusModalSetup />}*/}
```

Comment:
Old modal disabled.
Re-enable when updated with new format.
Will need to make it aware of the progress bar somehow, to not pop up when it's open.